README.md

```diff
@@ -146,6 +146,17 @@ pip3 install -r sphinx/requirements.txt
 python3 -m sphinx -T -E -b html -d _build/doctrees -D language=en . _build/html
 ```
 
+### Improving performance on MI300 when using less than 8 GPUs
+
+On a system with 8\*MI300X GPUs, each pair of GPUs are connected with dedicated XGMI links in a fully-connected topology. So, for collective operations, one can achieve good performance when all 8 GPUs (and all XGMI links) are used. When using less than 8 GPUs, one can only achieve a fraction of the potential bandwidth on the system.
```

Comment:
Is it "good" performance, or "the best possible performance" or "optimal"? "Good" is very vague. A more precise term would be helpful (but not necessary).

> On a system with 8\*MI300X GPUs, each pair of GPUs is connected to dedicated XGMI links in a fully-connected topology. For collective operations, good performance is achieved when all 8 GPUs and all XGMI links are used. When using fewer than 8 GPUs, only a fraction of the potential bandwidth on the system can be achieved.